src/SwiftPackage.ts

```diff
@@ -45,39 +45,55 @@ export interface Dependency {
     url?: string
 }
 
+// package we attempted to load but failed
+class InvalidPackage implements PackageContents {
```

Comment:
Do you intend to use this class by name? If not, we don't really need a class or something that explicitly implements `PackageContents`. We could just return:

```javascript
{
  name: '',
  products: [],
  dependencies: [],
  targets: []
}
```

from `loadPackage`. That will also conform to `PackageContents`.

Reply:
Just added SwiftPackage.isValid to differentiate between packages that loaded and failed to load